packages/scalecube-services/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@scalecube/scalecube-services",
+  "description": "Scalecube-js provides microservices pattern, in easy to use way. Supporting NodeJS and Browers (your client doesn't have to be mono!)",
+  "version": "0.0.6",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/scalecube/scalecube-js/packages/scalecube-services"
+  },
+  "main": "build/index.js",
+  "#module": "es/index.js",
```

Comment:
why here `module` with `#` ?

Reply:
it was in the main package.json in the same way, I just dublicated it for each package.